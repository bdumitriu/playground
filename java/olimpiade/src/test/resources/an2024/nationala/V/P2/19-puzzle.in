2 100 2
23964 22141 13831 27599 13374 32461 13455 32359 31319 31116 25742 25747 28822 6712 26577 2413 6869 921 13129 25456 13589 26659 16445 22617 19215 88 18321 11664 26843 22779 26351 5771 7369 6451 23635 13624 16591 1444 16651 28435 2678 13716 15152 14491 13619 24238 1374 7576 3832 17564 23761 6836 28595 7728 437 7768 23943 32633 21415 11143 29646 27255 6768 1874 29981 7458 24728 21648 2592 3829 4596 18364 21217 1835 4923 15251 19143 13793 15991 8241 13843 5275 12473 2423 1539 2313 13164 399 14383 22121 1331 26542 17243 3243 1534 117 14342 16422 18333 1764
