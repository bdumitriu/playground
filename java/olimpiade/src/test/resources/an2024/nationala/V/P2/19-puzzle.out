17566836 74581648
